### ISSUE TITLE
[Bug] iOS large title doesn't clear background

### ISSUE THREAD

**Issue by breyed** [xamarin/Xamarin.Forms#7679]
When a `NavigationPage` has large titles enabled (`SetPrefersLargeTitles(navigationPage.On<iOS>(), true)`), the page transition animation shows the large title background as gray. After the navigation completes, the background changes to white. Additionally, if the prior page has a `ListView` with groups, the group headers continue to be visible on top of the large title during the animation. Each visual artifact makes the transition look janky.

This is a regression. I first encountered this bug after upgrading to XF 4.2.0.815419 and iOS 13.

### Basic Information

- Version with issue: 4.2.0.815419
- Last known good version: 4.0?
- IDE: VS 16.3.0 (Windows), VS 3.8 (build 1805) (Mac), Xcode 11
- Platform Target Frameworks:
  - iOS: 13.1

### Reproduction

ControlGallery => NavigationBar Gallery -Legacy => Go to Searchbar TitlePage

**Comment by samhouts** [comment]
@breyed  Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by breyed** [comment]
@samhouts I don't have an isolated repro for this bug. It shows up in a production app that isn't doing anything special with its large titles. I expect at least the gray background issue will readily repro for you on an existing test project using the latest XF and iOS that uses large titles. Fixing the background bug will automatically fix the text overlay bug. If you have trouble reproing, let me know.

**Comment by PureWeen** [comment]
Some related issues

https://github.com/xamarin/Xamarin.Forms/issues/7701
https://github.com/xamarin/Xamarin.Forms/issues/7350

**Comment by ianthetechie** [comment]
I have noticed this issue as well. This will likely delay shipping of our app if it isn't fixed soon as this looks awful.

**Comment by jfversluis** [comment]
There is a PR attached to this issue that possibly fixes it. While we are processing that, you might be able to look at the code and turn it into a solution useable through a custom renderer so you won't be blocked.

**Comment by ianthetechie** [comment]
Ooh, fantastic. Thanks; I'll look into that.

**Comment by ianthetechie** [comment]
Any movement on this? I've seen several XF releases come and go with no movement.

**Comment by ianthetechie** [comment]
Bump? Is this not scheduled for any release? Or am I missing something?

**Comment by james1301** [comment]
@samhouts any feedback on @ianthetechie's comment?

**Comment by samhouts** [comment]
There was a snag on the PR. It requires that our builds move to VS 2019 to use the new iOS13 APIs. However, there's current a compatibility issue with VS 2017 if we do that. We're working to resolve that as soon as possible. Thank you for your patience.

**Comment by breyed** [comment]
@samhouts If you aren't already doing so, could you in the process address compatibility between VS versions generally? Compatibility with iOS releases should be tested and addressed during the iOS beta phase, not after release, triggered by developers and customers reporting bugs. By the time a iOS update is finalized, it should "just work" if you're on the latest stable Xamarin bits. The last slew of half-a-dozen XF bugs to work around was a hit the productivity boost that XF is supposed to provide.

Addressing iOS update issues early in their lifecycle may mean working with beta versions of VS that target the new iOS versions. The Xamarin development process has to be ready to handle that while still working with the production version of VS.

**Comment by ianthetechie** [comment]
Yeah... After almost 2 years of Xamarin dev, a proposal I took some heat for, I'll never recommend anything but pure native again. The slow turnaround on bug fixes to simple views is astounding.

I understand the desire to retain compatibility with VS 2017, but that comes across as odd for two reasons. 1) Mobile developers (esp. iOS) always use the latest (or very close to it) dev tools. 2) Retaining backward compatibility is coming at the cost of leaving something broken in current OS versions. And it's not like this is a "stable" release scenario where you're slowing the release of new functionality; it's not fixing an issue in an API that's been around since iOS 11. (And yes, I know Apple screwed up this a bit in iOS 13 with new APIs, but clearly you have already worked around this).

TL;DR sacrificing UX for months because some devs won't update their copy of VS is a pretty odd decision to me. Sorry for the rant, and hope this is fixed soon.
